loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW

```diff
@@ -168,22 +194,79 @@ DEFINE
  VALIDCHRINPUT1            = CHARACTER
  VALIDCHARS                = NUMBER ARRAY(999)
  MAXCHRVALUE               = 255
-END
+
+[* 01/21/22 *]
+
+ APPROVALCODESEXCLUDED     = NUMBER ARRAY(9999)
+ BNOCFGFILENAME            = CHARACTER
+ BNODEBUGCOUNT             = NUMBER
+ BNODEBUGLOOP              = NUMBER
+ BNODEBUGMODE              = NUMBER
+ BNODEBUGMODEOFF           = 0
+ BNODEBUGMODEON            = 1
+ BNODEBUGMSG               = CHARACTER
+ BNODEBUGMSGLIST           = CHARACTER ARRAY(999)
+ BNODEBUGMSGMAX            = 999
+ BNOINCLUDEPARMINDEBUG     = NUMBER
+ BNOINDENT                 = CHARACTER ARRAY(9)
+ BNOINDENTMAX              = 9
+ BNOLASTMODDATE            = DATE
+ BNOLASTMODTIME            = CHARACTER
+ BNOLOCALLOOP              = NUMBER
+ BNOPRINTDEBUGDAYS         = NUMBER
+ BNOPRINTDEBUGINFODAYS     = NUMBER
+ BNOPROGRAMNAME            = CHARACTER
+ BNOPROGRAMVERSION         = CHARACTER
+ CUTOFFDQA                 = NUMBER
+ CUTOFFDQB                 = NUMBER
+ CUTOFFDQC                 = NUMBER
+ CUTOFFDQD                 = NUMBER
+ CUTOFFDQE                 = NUMBER
+ DUEDATECALCTYPE           = CHARACTER ARRAY(13)
+ ELEGIBLEFREQUENCIES       = NUMBER ARRAY(13)
+ INCLUDEPROGRAMINFO        = NUMBER
+ INELIGIBLECODEAPPROVALCODE= 13
+ INELIGIBLECODEDQPAYMENTS  = 14
+ INELIGIBLECODEFREQ        = 11
+ INELIGIBLECODEMINPAYMENTCOUNT=12
+ LNEWDUEMONTH              = NUMBER
+ MINIMUMPAYMENTCOUNT       = NUMBER
+ QQ                        = CHARACTER
+ TESTACCOUNTLIST           = CHARACTER
+ TESTMODE                  = NUMBER
+ TMPPOS                    = NUMBER
+ TMPVAL                    = NUMBER
+ TRUEFALSECHR              = CHARACTER ARRAY(1)
+ VALIDTESTACCOUNT          = NUMBER
+END [DEFINE]
 
 SETUP
  Q=CTRLCHR(34)
-
+ QQ=CTRLCHR(34)

```

Comment:
Why 2 vars with the same value?

Reply:
A carryover from 'borrowed' code from my templates.  corrected.